lib/broccoli/angular2-app.js

```diff
@@ -87,10 +87,16 @@ Angular2App.prototype.toTree = function () {
     destDir: 'vendor'
   });
 
+  var publicDirTree = new Funnel(path.join(sourceDir, 'public'), {
```

Comment:
Ok, makes more sense now. But just so I'm clear, where do you want `public` to live? :) In the root or `src`?